### PR TITLE
Update donation page

### PR DESCRIPTION
**Pull request by stvnrlly**
This switches from a link to the CFA donation page to an embedded form that'll deposit straight to the account managed by District Civic Labs. It uses Stripe and AWS Lambda to make things work behind the scenes.

The Lambda URL could use some DNS beautifying, but that can be included in this pull request or updated later.

### CODE REVIEW

File: donate.html [hunk 1]
```diff
@@ -5,12 +5,92 @@
...
+          // Send a request to our AWS Lambda function, which handles the
+          // secret Stripe stuff.
+          let send = JSON.stringify({
+            "token": token.id,
+            "amount": amount,
+            "instruction": instruction,
+            "email": token.email,
+            "account": account
+          });
+          $.post("https://b9sq87esw7.execute-api.us-east-2.amazonaws.com/prod/codefordc-stripe-processor", send, function(data, status){
```
[the remark this answers is not in the document]
Comment (stvnrlly) [reply donate.html]: No, the District Civic Labs one. We should discuss consolidation and/or access for you.

File: donate.html [hunk 2]
```diff
@@ -5,12 +5,92 @@
...
+        var handler = StripeCheckout.configure({
+        key: apiKey,
+        image: 'https://stripe.com/img/documentation/checkout/marketplace.png',
+        locale: 'auto',
+        token: function(token) {
+          // You can access the token ID with `token.id`.
+          // Get the token ID to your server-side code for use.
+          // The "00" is there because Stripe deals in cents rather than dollars, but the form
+          // only asks for a whole dollar amount.
+          var amount = Number(document.getElementById('donation-amount').value + "00")
```
[the remark this answers is not in the document]
Comment (stvnrlly) [reply donate.html]: I can handle that :)

File: donate.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkalish) [reply donate.html]: Cool, are you free to meet for a half hour this weekend next week to figure it out?

[the remark this answers is not in the document]
Comment (stvnrlly) [reply donate.html]: Yeah, there should be something available. I'll follow up with you.

**Pull request merged**